api/resources/translations/messages-en.properties

```diff
@@ -753,6 +753,7 @@ instance.upgrade = Upgrades
 instance.upgrade.at = On
 instance.upgrade.betas = Betas
 instance.upgrade.branches = Branches
+instance.upgrade.build.version = Build
```

Comment:
Maybe `Build number` would be more obvious?

Reply:
"Build" seems fine to me (especially since we say "Version" and not "Version number")